src/gui/lazyfilelist.h

```diff
@@ -135,7 +138,7 @@ class LazyDirView {
                     if (strcmp(files[0].lfn, dir.fno.fname) != 0) {
                         // i.e. we didn't get the same entry as the zeroth entry (which may occur when populating the window with non-null firstDirEntry)
                         // Make place in the window by standard item rotation downwards (to the right)
-                        std::rotate(files.rbegin(), files.rbegin() + 1, std::make_reverse_iterator(i)); // solves also the case, when there are less files in the window
+                        std::rotate(files.rbegin(), files.rbegin() + 1, std::reverse_iterator<typename decltype(files)::iterator>(i)); // solves also the case, when there are less files in the window
```

Comment:
I'd really like to move to C++14 ... if not, I'll make my own make_reverse_iterator, this statement looks horrible

Reply:
It does look horrible, I agree 😅. Fixed.